apps/meteor/server/services/federation/infrastructure/rocket-chat/hooks/index.ts

```diff
@@ -48,6 +47,9 @@ export class FederationHooks {
 				if (!params?.user || !room) {
 					return;
 				}
+
+				throwIfFederationNotEnabledOrReady();
```

Comment:
I think we should not validate this here because we run only validations here (e.g. someone trying to add a federated user to a non-federated room), that's why we were not checking if federation was enabled.